File: coco/src/main/java/coco/ep/m7i2/Divd.java
package coco.ep.m7i2;

import coco.ep.Exp;

public interface Divd<FT> extends coco.ep.m7.Divd<FT>, coco.ep.m3.Divd<FT>, coco.ep.m7i2.Exp<FT>, Factory<FT> {

    default Exp<FT> multby(Exp<FT> other) {
        return this.mult(this, convert(other));
    }
    default Exp<FT> powby(Exp<FT> other) {
        return this.power(this, convert(other));
    }
}

File: coco_csharp/CoCo/EP/M7I2/Divd.cs
namespace EP.M7I2 {
    public interface Divd<FT> : Exp<FT>, EP.M7.Divd<FT> {
        EP.Exp<FT> EP.I1.Exp<FT>.multby(EP.Exp<FT> other) {
            return mult(this, convert(other));
        }
        EP.Exp<FT> EP.M7.Exp<FT>.powby(EP.Exp<FT> other) {
            return power(this, convert(other));
        }
    }
}